### PR TITLE
Add .env.example file and update README.md

### PR DESCRIPTION
**Pull request by KBandipo**
Add .env.example and update README.md

### CODE REVIEW

**Comment by KBandipo** [comment]
PR ready for review @collabo-community/review-web-app @Ifycode

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# .env.example
+
+# GitHub API Token
```
Comment (Ifycode) [inline .env.example:3]: Question is this description of the token "GitHub API token" correct?
What type of github token is the user to use here: 
- User access token?
- Personal access token?
- Github API token?

You want to state it exactly as it is to not leave a user/contributor confused.

Advising that you go to the place where you generated the token from and update this description accordingly @KBandipo

**Review by Ifycode — CHANGES_REQUESTED** [review]

**Comment by Ifycode** [comment]
PR title also just says "Update", so what exactly are you updating? @KBandipo

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+GITHUB_TOKEN=your_github_token_here
```
[the remark this answers is not in the document]
Comment (KBandipo) [reply .env.example]: Done

File: .env.example (hunk 1 above)
Comment (KBandipo) [reply .env.example]: Done

**Comment by KBandipo** [comment]
Pull Request ready for review @collabo-community/review-web-app @Ifycode

File: .env.example (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ifycode) [reply .env.example]: You can make this part better desciptive as I have explained in the "Also note" part @KBandipo and change it in everywhere else it appears in the code.

**Comment by Ifycode** [comment]
See comment here: https://github.com/collabo-community/collabocate/pull/1#discussion_r1646313107
@KBandipo

File: .env.example (hunk 2 above)
[the remark this answers is not in the document]
Comment (KBandipo) [reply .env.example]: Okay, I'll do that now

**Comment by KBandipo** [comment]
I have made the requested changes, please review the updates @collabo-community/review-web-app @Ifycode

**Review by Ifycode — APPROVED** [review]

**Pull request merged**